modules/theme/src/themes/default/elements/input.overrides

```diff
@@ -83,5 +83,11 @@
         .copy-input-eye-icon {
             margin-right: @copyInputFieldMarginRight;
         }
+        &.copy-input-dark {
```

Comment:
Let's get rid of this in a different PR since it was not used. @vihanga-liyanage

Reply:
We decided to go with black. So this us used in extensions.